app/views/shared/forms/_identify_types_of_outgoings_form.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= form_with(
+      model: @legal_aid_application,
+      url: form_path,
+      method: :patch,
+      local: true
+    ) do |form| %>
+
+  <%= govuk_form_group show_error_if: @legal_aid_application.errors.present? do %>
+    <%= govuk_fieldset_header do %>
+      <h1 class="govuk-fieldset__heading govuk-!-padding-bottom-7"><%= page_title %></h1>
+      <h2 class="govuk-heading-m"><%= t('.sub_heading') %></h2>
+    <% end %>
+
+    <div class="deselect-group" data-deselect-ctrl="#none_selected">
+      <%= transaction_type_check_boxes(
+            form: form,
+            transaction_types: TransactionType.debits
+          ) %>
+    </div>
+
+    <p class="govuk-!-padding-top-4"><%= t('.or_break') %></p>
+
+    <div class="govuk-checkboxes">
+      <div class="govuk-checkboxes__item">
+        <%= check_box_tag :none_selected, 'true', false, class: 'govuk-checkboxes__input' %>
+        <%= label_tag :none_selected, t('.none_selected'), class: 'govuk-label govuk-checkboxes__label' %>
+      </div>
+    </div>
+
+  <% end %>
+
+  <%= next_action_buttons(
+        show_draft: local_assigns.key?(:show_draft) ? show_draft : false,
```

Comment:
Hmmm. Perhaps that's the way I should have done it in my PR. I've created a helper method that checks which journey we're in, and uses that to determine whether the draft button is required: `show_draft: current_journey?(:providers)`.

Reply:
I think both approaches are good. Maybe yours is better actually...
You also had to deal with locales different depending on the journey.
I used this pattern because it is used everywhere else:
![image](https://user-images.githubusercontent.com/482806/58162810-c2f1ed80-7c7a-11e9-933b-c72c7f53af11.png)

Reply:
Keep yours the way it is. I'll update mine to match the current code base.